configure-places-in-the-sdk/places-extension/using-places-without-active-region-monitoring.md

```diff
@@ -0,0 +1,415 @@
+            return;
+        }
+
+        Location currentLocation = result.getLastLocation();
+        if (currentLocation == null) {
+            return;
+        }
+
+        // ask the Places SDK for the 10 nearest Points of Interest based on the user's location
+        Places.getNearbyPointsOfInterest(currentLocation, 10,
+                new AdobeCallback<List<PlacesPOI>>() {
+                    @Override
+                    public void call(List<PlacesPOI> pois) {
+                        // pois is the 10 nearest POIs based on the location
+                    }
+                }, new AdobeCallback<PlacesRequestError>() {
+                    @Override
+                    public void call(PlacesRequestError placesRequestError) {
+                        // Look for the placesRequestError and handle the error accordingly
+                    }
+                });
+    }
+}
+```
+{% endtab %}
+
+{% tab title="iOS (Objective-C)" %}
+
+Example implementation from a [`CLLocationManagerDelegate`](https://developer.apple.com/documentation/corelocation/cllocationmanager?language=objc) method [`locationManager:didUpdateLocations:`](https://developer.apple.com/documentation/corelocation/cllocationmanagerdelegate/1423615-locationmanager?language=objc):
+
+```objectivec
+- (void) locationManager:(CLLocationManager*)manager didUpdateLocations:(NSArray<CLLocation*>*)locations {
+    // ask the Places SDK for the 10 nearest Points of Interest based on the user's location
+    [ACPPlaces getNearbyPointsOfInterest:[locations lastObject] limit:10 callback:^(NSArray<ACPPlacesPoi *> * _Nullable nearbyPoi) {
+        // nearbyPoi is the 10 nearest POIs based on the location
+    } errorCallback:^(ACPPlacesRequestError result) {
+        // log the error if we got one
+        NSLog(@"error: %lu", (unsigned long)result);
+    }];
+}
+```
+{% endtab %}
+
+{% tab title="iOS (Swift)" %}
+
+Example implementation from a [`CLLocationManagerDelegate`](https://developer.apple.com/documentation/corelocation/cllocationmanager) method [`locationManager(_:didUpdateLocations:)`](https://developer.apple.com/documentation/corelocation/cllocationmanagerdelegate/1423615-locationmanager):
+
+```swift
+func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
+    // ask the Places SDK for the 10 nearest Points of Interest based on the user's location
+    ACPPlaces.getNearbyPoints(ofInterest: locations.last!, limit: 10, callback: { (nearbyPoi) in
+        // nearbyPoi is the 10 nearest POIs based on the location
+    }) { (error) in
+        // log the error if we have one
+        print("error: \(error)")
+    }
+}
+```
+{% endtab %}
+
+{% endtabs %}
+
+## Trigger Entry Events When the User is in a POI
+
+The SDK will give you back a list of nearby POIs, including whether or not the user is currently within each of the POIs. If the user is in a POI, you can have the SDK trigger an entry event for that region.
```

Comment:
- Please reword to: "The SDK returns a list of..."
- Please remove "or not"
Please reword "currently within each of the POIs" to "currently within each POI."